examples/subscribe.rs

```diff
@@ -147,9 +130,11 @@ async fn main() -> Result<(), Box<dyn snafu::Error>> {
     // drop(subscription_clone);
 
```

Comment:
Very useful comments <3
That's very important that we unsubscribe on drop.

Reply:
We do actually :)